tests/testaudience.js

```diff
@@ -1,21 +1,36 @@
+/**
+ * @fileoverview Defines tests for audience engagement.
+ *
+ * @see http://google.github.io/styleguide/javascriptguide.xml
+ * @see http://developers.google.com/closure/compiler/docs/js-for-compiler
+ */
 
 
+/** @const {!RegExp} */ var PATTERN = /^\d([,\.:]*\d)*[%km]?$/;
```

Comment:
If page views is "1g" (one billion)?
See [`formatters.NumberFormatter.roundNumber`](https://github.com/Datamart/Glize/blob/master/src/formatters/NumberFormatter.js#L57) for more details.